qemu/tests/block_iscsi_with_specical_max_sectors.py

```diff
@@ -0,0 +1,106 @@
+"""Test IO on specific max_sector_kb of disk """
+
+import logging
```

Comment:
we may need test.log instead, for logging cannot print out the messages correctly, refer to VT patch https://github.com/autotest/tp-qemu/issues/3048 for the details

Reply:
updated